docs/docs/default-actions.mdx

```diff
@@ -148,3 +148,17 @@ their message.
 This action undoes the last user-bot interaction. It can be triggered by the user
 by sending a "/back" message to the assistant if the [RulePolicy](./policies.mdx#rule-policy) is configured.
                                                                                                                                                                                                                                                                         |
+## Form Action
+
+By default Rasa Open Source uses the inbuilt `FormAction` for processing any
+[form logic](forms.mdx). You can override this default action with a custom action by
+adding a custom action with the form's name to the domain. This can e.g. be used to use
+the deprecated `FormAction` which is part of the Rasa SDK and used to implement forms
+for Rasa Open Source 1.
```

Comment:
```suggestion
By default Rasa Open Source uses `FormAction` for processing any
[form logic](forms.mdx). You can override this default action with a custom action by
adding a custom action with the form's name to the domain. This can e.g. be used to use
the deprecated `FormAction` which is part of the Rasa SDK and used to implement forms
for Rasa Open Source 1.
```

Reply:
@wochinge didn't I say that we should just remove the "caution" note here and just state in the text that overriding should ONLY be done during migration? So basically say this directly in the text, rather than saying "This can e.g. be used".

Reply:
Yes, but I haven't worked that in yet.

Reply:
@melindaloubser1 Merged your suggestion with Akela's

Reply:
haha ok sorry, i assumed since you requested Melinda for review you weren't going to ;)